asmcli/commands/validate.sh

```diff
@@ -119,6 +119,14 @@ validate_hub() {
   if ! is_managed && [[ "${USE_VM}" -eq 1 && "${USE_HUB_WIP}" -eq 0 ]]; then
     fatal "Hub Workload Identity Pool is required to add VM workloads. Run the script with the -o hub-meshca option."
   fi
+
+  if ! is_managed && [[ "${CA}" == "mesh_ca" && "${USE_HUB_WIP}" -eq 0 ]] && ( can_register_cluster && is_gcp || is_cluster_registered ) ; then
+    info "Fleet workload identity pool is used as default for Mesh CA. "
+    context_set-option "USE_HUB_WIP" 1
+    local OPTIONAL_OVERLAY; OPTIONAL_OVERLAY="$(context_get-option "OPTIONAL_OVERLAY")"
+    OPTIONAL_OVERLAY="hub-meshca,${OPTIONAL_OVERLAY}"
```

Comment:
Is there anything in this overlay that's still necessary that's not already included with MeshCA?

Reply:
You mean not covered by https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/blob/main/asm/istio/istio-operator.yaml? There are some setters which have different behavior, especially when it is multiproject. We can merge the istio-operator together with hub-meshca.yaml. In this case, some UG needs to be changed like the gke multiproject installation guide since setters will be changed in the istio operator. Maybe we can do this merge in 1.11 and hopefully at that time gke multiproject installation guide will be retired?